cli/export/export.go

```diff
@@ -1,27 +1,40 @@
 /*
 Tool to export annotation results from the internal DB to an output DB.
+It does a simple copy&paste of the internal DB. The annotation results are
+stored in the assignments table.
 
-Usage: export <path-to-origin.db> <path-to-destination.db>
+Usage: export DSN DSN
```

Comment:
Is DSN better then DB_URL ?

Also related - the requirement on the UI level for export is: 
 - input: url to point to inernal DB
 - output: path in the filesystem to sqlight DB

And viasurvesa for import.

I believe it has advantage of simplicity for this app the user, at the same time agility for its developers (choose any DB)

Could you please update it?

Reply:
imo DSN is better. It's standard name for it. If I read DB_URL, I'm not sure what format should be used.
For other points 👍

Reply:
👍  a good argument for using DSN for DB part also might be posting something like https://en.wikipedia.org/wiki/Data_source_name :)

Reply:
there even a wiki page for it! I just saw it across documentation of different drivers & tools.

Reply:
@bzz:

> input: url to point to inernal DB
> output: path in the filesystem to sqlight DB

Please confirm: you want the output argument to accept only sqlite:// strings, or do you want it to ask for a path? i.e.

1. `export postgresql://user:password@host:port sqlite:///path/to/db.db`
2. ` export postgresql://user:password@host:port /path/to/db.db`

Reply:
2 in my opinion follows the right contract.
Import would be `import /path/to/db.db postgresql://user:password@host:port `